modules/backing-up-central-database.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * cli/managing-central-database.adoc
+:_content-type: PROCEDURE
+[id="backing-up-central-database_{context}"]
+= Backing up Central database
+
+.Procedure
+
+* Run the following command to back up Central's database:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl "$ROX_CENTRAL_ADDRESS" central backup
```

Comment:
The correct variable is `ROX_ENDPOINT` and we do not need to set this explicitly here.

Since we initially documented it together with the `ROX_API_TOKEN` we can drop this here.

If we want to keep it, let's use it like:
```bash
export ROX_ENDPOINT=<host:port>
roxctl image check --image=<image name>
```